sentry-servlet-jakarta/src/main/java/io/sentry/servlet/jakarta/SentryServletRequestListener.java

```diff
@@ -38,15 +38,16 @@ public SentryServletRequestListener() {
   @Override
   public void requestDestroyed(@NotNull ServletRequestEvent servletRequestEvent) {
     final ServletRequest servletRequest = servletRequestEvent.getServletRequest();
-    LifecycleHelper.close(servletRequest.getAttribute(SENTRY_LIFECYCLE_TOKEN_KEY));
+    LifecycleHelper.close(servletRequest.getAttribute(SENTRY_SCOPE_LIFECYCLE_TOKEN_KEY));
   }
 
   @Override
   public void requestInitialized(@NotNull ServletRequestEvent servletRequestEvent) {
-    final @NotNull ISentryLifecycleToken lifecycleToken = scopes.pushIsolationScope();
+    final @NotNull ISentryLifecycleToken lifecycleToken =
```

Comment:
Can't use `try` or `with` here since we're passing the lifecycle token via attributes. Same is true for some other locations in this PR.